tools/enter-data-helper/src/components/FlavorOverview.js

```diff
@@ -62,11 +62,22 @@ class SubtypeOverview extends JsonFormsContainer {
         this.state.schema = JSON.parse(fs.readFileSync(path.join(ROOT_DIR, 'schema',
             { 'bench': 'benchmark', 'param': 'paramset', 'impl': 'implementation' }[this.subType] + '.json')));
         this.state.uiSchema = Generate.uiSchema(this.state.schema);
+        if (this.subType === 'bench') {
+            var parts = this.subName.split('_');
+            this.dataStore.impl = parts[0];
```

Comment:
Are you sure `parts` has two elements? :slightly_smiling_face:

Reply:
If it hasn't then the filename is not valid and validate.sh fails. So, working as intended I guess :smiling_imp: